### DISCUSSION TITLE
Export configuration/settings

### DISCUSSION THREAD

**Discussion by matthewm1970** [maruohon/minihud#336] [Ideas]
I play on multiple different servers, from two separate computers, and occasionally also need to log into worlds to test concepts and so on.  I find it very tedious to re-create all my preferred settings within MiniHUD but it's so valuable I do it anyway each time I add a new config or world to a new computer or instance (I use the Prism launcher) because I don't find it comfortable to play without it any more. It's amazing to me that this isn't part of the vanilla client.

I'd love some way to export my settings and selections from one instance of miniHUD to be imported and immediately used in another instance!

**Comment by sakura-ryoko** [comment]
You can make a copy of your config files, (Vanilla Launcher is %APPDATA%\.minecraft\config) and put them on the second machine.
